libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.WebApi/BotProactiveMessageHandler.cs

```diff
@@ -43,9 +43,9 @@ public BotProactiveMessageHandler(BotFrameworkAdapter botFrameworkAdapter)
                 throw new InvalidOperationException($"Expected a Bot App ID in a header named \"{BotAppIdHttpHeaderName}\" or in a querystring parameter named \"{BotAppIdQueryStringParameterName}\".");
             }
 
-            var conversationReference = await request.Content.ReadAsAsync<ConversationReference>(BotMessageHandlerBase.BotMessageMediaTypeFormatters, cancellationToken);
+            var conversationReference = await request.Content.ReadAsAsync<ConversationReference>(BotMessageHandlerBase.BotMessageMediaTypeFormatters, cancellationToken).ConfigureAwait(false);
```

Comment:
See earlier comment on not using `.ConfigureAwait(false)` in ASP.NET specific components.